pkgs/applications/blockchains/btcpayserver/update.sh

```diff
@@ -8,4 +8,4 @@ echo "Updating nbxplorer"
 ../nbxplorer/update.sh
 echo
 echo "Updating btcpayserver"
-../nbxplorer/util/update-common.sh btcpayserver deps.nix
+../nbxplorer/util/update-common.sh btcpayserver deps.nix '"/p:Configuration="Altcoins-Release" /p:RazorCompileOnBuild=false"'
```

Comment:
Same here - why is `RazorCompileOnBuild=false` needed?

Reply:
Needed, see https://github.com/NixOS/nixpkgs/pull/130162/commits/d17ebc3252fa522ee9e52e61b29501c5f231c32e